3.4.1/scala2.12-java11-ubuntu/entrypoint.sh

```diff
@@ -77,6 +77,9 @@ elif ! [ -z "${SPARK_HOME+x}" ]; then
   SPARK_CLASSPATH="$SPARK_HOME/conf:$SPARK_CLASSPATH";
 fi
 
+# SPARK-43540: add current working directory into executor classpath
```

Comment:
This is 3.5.x series, we might create a `entrypoint.sh.3.4.template`, `Dockerfile.3.4.template`.

We also [change](https://github.com/apache/spark-docker/blob/master/add-dockerfiles.sh#L53) the `add-dockerfiles.sh`, For 3.x.1 version, if `3.x` templates exists, use 3.x template, otherwise use the master entrypoint and Dockerfile directly.

It can be a separate PR.

Reply:
cc @HyukjinKwon @zhengruifeng @dongjoon-hyun 

Or we could just apply master dockerfile changes in 3.4-branches?

Reply:
I think creating a template is better.